go.mod

```diff
@@ -81,5 +81,5 @@ replace (
 replace (
 	github.com/openshift-kni/performance-addon-operators => github.com/openshift-kni/performance-addon-operators v0.0.0-20200805211725-320b53af2928 // master
 	github.com/openshift/ptp-operator => github.com/openshift/ptp-operator v0.0.0-20200626140756-02c7317c2dbd // release-4.6
-	github.com/openshift/sriov-network-operator => github.com/openshift/sriov-network-operator v0.0.0-20200722065212-ebf7a24cc54f // release-4.6
+	github.com/openshift/sriov-network-operator => github.com/openshift/sriov-network-operator v0.0.0-20200820150534-75dd3c1da73d // release-4.6
```

Comment:
shouldnt L26 in this file be changed too?
wont it be better if everything uses the same version of sriov-network-operator?

Reply:
that line is added automatically and is not used because this section is a replace section and will override the require section